packages/stateless/components/apps/AppsRenderer.tsx

```diff
@@ -64,121 +70,63 @@ export type AppsRendererProps = {
   chainPicker: ReactNode
 }
 
-export const AppsRenderer = ({
-  iframeRef,
-  fullScreen,
-  setFullScreen,
-  executionType,
-  setExecutionType,
-  otherAddress,
-  setOtherAddress,
-  AddressInput,
-  chainPicker,
-}: AppsRendererProps) => {
-  const [url, setUrl] = useQuerySyncedState({
-    param: 'url',
-    defaultValue: '',
-  })
+// Only allow URLs starting with `http(s)://`, to prevent XSS via `javascript:`
+// URLs.
+const ALLOWED_URL_REGEX = /^https?:\/\/.+[^\.]$/
 
-  let urlValid = false
+const isUrlValid = (url: string): true | string => {
   try {
-    urlValid = !!url && !!new URL(url).href && ALLOWED_URL_REGEX.test(url)
-  } catch {
-    // Ignore.
-  }
-
-  // Set full screen first time when URL is set. If the user manually closes the
```

Comment:
This automatic fullscreen feature was pretty useful to have.
I currently have a "view as team" feature on my website that will direct people to the apps tab of their selected DAO with their current URL. Removing this feature could be confusing for new users since they have to process the new layout and eventually click open app.

Reply:
ah yes ok, added it back 👍